src/test/java/test/integration/AddArquillianAlgeronConsumerTest.java

```diff
@@ -0,0 +1,87 @@
+import org.jboss.forge.addon.projects.ProjectFactory;
+import org.jboss.forge.addon.projects.facets.DependencyFacet;
+import org.jboss.forge.addon.ui.controller.CommandController;
+import org.jboss.forge.addon.ui.result.Failed;
+import org.jboss.forge.addon.ui.result.Result;
+import org.jboss.forge.addon.ui.test.UITestHarness;
+import org.jboss.forge.arquillian.command.AddAlgeronConsumerDependenciesCommand;
+import org.jboss.forge.arquillian.command.AddArquillianCommand;
+import org.jboss.forge.arquillian.command.AddTestFrameworkCommand;
+import org.jboss.forge.arquillian.container.model.ContractConsumerLibrary;
+import org.jboss.forge.arquillian.testframework.algeron.AlgeronConsumer;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.inject.Inject;
+
+@RunWith(Arquillian.class)
+public class AddArquillianAlgeronConsumerTest
+{
+
+   @Inject
+   private UITestHarness testHarness;
+
+   @Inject
+   private ProjectFactory factory;
+
+   @Test
+   public void should_add_arquillian_algeron_consumer_dependencies() throws Exception {
+
+      Project project = factory.createTempProject();
+
+      try(CommandController addCommandController = testHarness.createCommandController(AddArquillianCommand.class, project.getRoot())) {
+         addCommandController.initialize();
+         addCommandController.setValueFor("arquillianVersion", "1.0.0.Alpha5");
+         Result result = addCommandController.execute();
+
+         checkResult(result);
+      }
+      project = factory.findProject(project.getRoot());
+
+      try(CommandController addTestFrameworkCommandController = testHarness.createCommandController(AddTestFrameworkCommand.class, project.getRoot())) {
+         addTestFrameworkCommandController.initialize();
+         addTestFrameworkCommandController.setValueFor("standalone", true);
+         addTestFrameworkCommandController.setValueFor("testFramework", "junit");
+         Result result = addTestFrameworkCommandController.execute();
+
+         checkResult(result);
+
+      }
+
+      try(CommandController addAlgeronConsumerCommandController = testHarness.createCommandController(AddAlgeronConsumerDependenciesCommand.class, project.getRoot())) {
+         addAlgeronConsumerCommandController.initialize();
+         addAlgeronConsumerCommandController.setValueFor("type", "pact");
+         Result result = addAlgeronConsumerCommandController.execute();
+
+         checkResult(result);
+
+         Assert.assertTrue(project.hasFacet(AlgeronConsumer.class));
+
+         final DependencyFacet facet = project.getFacet(DependencyFacet.class);
+         Assert.assertTrue(facet.hasDirectDependency(ContractConsumerLibrary.PACT.getAlgeronConsumer()));
+         Assert.assertTrue(facet.hasDirectDependency(ContractConsumerLibrary.PACT.getContractConsumer()));
+
+      }
+
+   }
+
+   private void checkResult(Result result) {
+      Assert.assertFalse(result instanceof Failed);
+      printIfFailed(result);
```

Comment:
I think this line will never be reached since the assertion above throws ISE if that statement is true?